internal/runners/packages/packages.go

```diff
@@ -127,6 +127,28 @@ func executePackageOperation(pj *project.Project, out output.Outputer, authentic
 	return nil
 }
 
+func addSuggestions(err error, ns model.Namespace, name string) error {
+	results, searchFail := model.SearchIngredients(ns, name)
+	if searchFail != nil {
+		// Log and return generic error if search failed
+		logging.Error("Failed to search for ingredients with namespace: %s and name: %s, got error: %v", ns.String(), name, searchFail)
+		return locale.WrapError(err, "package_ingredient_err_search", "Failed to resolve ingredient named: {{.V0}}", name)
+	}
+
+	maxResults := 5
+	if len(results) > maxResults {
+		results = results[:maxResults]
+	}
+
+	suggestions := make([]string, maxResults)
+	for i := range results {
+		suggestions[i] = fmt.Sprintf(" - %s", *results[i].Ingredient.Name)
```

Comment:
Should just use a slice of strings to print these. The plain outputter will translate that into a readable list.